unit_test/Makefile

```diff
@@ -2,7 +2,7 @@
 
 CC         := gcc
 INCLUDE    := -I../kernel/include
-CFLAGS	   := -pipe -nostdlib -nostdinc -ffreestanding -fms-extensions
+CFLAGS	   := -pipe -nostdlib -nostdinc -ffreestanding -fms-extensions -Wno-int-to-pointer-cast

```

Comment:
Remove because we return NULL instead of -1 for addresses and offsets

Reply:
In `stack.c`:
```c
for (int i = 0; i < NUM_PTB; i++) {
   ptb_stack[i] = (pte32_t*)(PTB_START + i*PTB_SIZE);
}
```
This causes an Wint-to-pointer-cast, thus not my problem